puppet/modules/consul/templates/consul.service.erb

```diff
@@ -0,0 +1,19 @@
+[Unit]
```

Comment:
The whole file needs to be parametrized

Reply:
https://github.com/jetstack/tarmak/blob/71e9b872b3d98adc0b4b08f428192020129fb4bb/puppet/modules/consul/templates/consul.service.erb